modules/extensions/autoscaler.tf

```diff
@@ -12,7 +12,7 @@ locals {
   cluster_autoscaler_enabled = alltrue([
     var.cluster_autoscaler_install,
     var.expected_node_count > 0,
-    length(local.worker_pools_autoscaling) > 0,
+    var.expected_autoscale_worker_pools > 0,
```

Comment:
Does this still work in the `null` case for this variable from `one()` in module-extensions.tf?

Reply:
You're right, we better use `coalesce` when we pass the value in `module-extensions.tf`.